src/wavm/WAVMWasmModule.cpp

```diff
@@ -851,11 +858,12 @@ bool WAVMWasmModule::execute(faabric::Message& msg, bool forceNoop)
     }
 
     // Set up OMP
-    setUpOpenMPContext(msg);
+    threads::setUpOpenMPContext(msg);
 
     // Executes OMP fork message if necessary
     if (msg.ompdepth() > 0) {
-        return executeAsOMPThread(msg);
+        // TODO - reimplement this call to execute a remote OpenMP thread
+        throw std::runtime_error("Not yet implemented remote OpenMP");
```

Comment:
Turns out this branch wasn't getting executed at all in tests any more. I'll resurrect as part of the next phase on cross-host OpenMP